demos/README.md

```diff
@@ -2,6 +2,8 @@
 
 ## Build instructions
 
+### Without CMake
```

Comment:
ISTM this defeats the point of adding CMake if we have to also keep separate build instructions up to date. And I suspect they will fairly rapidly fall out of date. So my belief is that this section should be deleted. WDYT @mmdriley ?

Reply:
Shall I remove the old build details ?